ietf-microwave-radio-link.yang

```diff
@@ -428,20 +398,29 @@ module ietf-microwave-radio-link {
         reference
           "ETSI EN 302 217-1";
         leaf maximum-nominal-power {
-          type power {
+          type mw-types:power {
             range "-99..99";
           }
           units "dBm";
           mandatory true;
           description
-            "Selected maximum output power.  Minimum output
-             power is the same as the system capability,
-             minimum-power.";
+            "Maximum output power.";
           reference
             "ETSI EN 302 217-1";
         }
+        leaf minimum-nominal-power {
+          type mw-types:power {
+            range "-99..99";
+          }
+          units "dBm";
+          mandatory true;
+          description
+            "Minimum output power.  By default, minimum output
+             power is the same as the system capability
+             minimum-power.";
```

Comment:
Should we capture the concept of nominal?

What about splitting the two statements?

```suggestion
            "Minimum output power.

             By default, minimum output power is the same as the
             system capability minimum-power.";
```

Reply:
I suggest adding this an issue and discussing on a call.